lib/board.rb

```diff
@@ -22,11 +22,18 @@ def initialize
               }
   end
   # looping through - nested loop for expansion
+  # should the methods below be a placement class?
```

Comment:
Saving this question for once all board class is complete so we're thorough on how we'd decide to create this, if needed.

Reply:
That makes sense! I also had a follow thought that maybe these methods could potentially live in the turn class? Since this information might be most relevant.

Reply:
Ooo interesting! Good conversation around the Turn class! Let's definitely think that through as we're planning out what that class is responsible for.